tests/operator_test.go

```diff
@@ -1195,6 +1195,56 @@ spec:
 		})
 	})
 
+	Describe("[test_id:6987]should apply component configuration", func() {
+
+		It("test VirtualMachineInstancesPerNode", func() {
+			newVirtualMachineInstancesPerNode := 10
+
+			By("Updating KubeVirt Object")
+			kv, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Get(originalKv.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(kv.Spec.Configuration.VirtualMachineInstancesPerNode).ToNot(Equal(&newVirtualMachineInstancesPerNode))
+			kv.Spec.Configuration.VirtualMachineInstancesPerNode = &newVirtualMachineInstancesPerNode
```

Comment:
Why is this check necessary?

Reply:
I refer here to `			Expect(kv.Spec.Configuration.VirtualMachineInstancesPerNode).ToNot(Equal(&newVirtualMachineInstancesPerNode))
`

Reply:
This is done to ensure that `newVirtualMachineInstancesPerNode`, which is hardcoded here to 10, is different than the default value of `kv.Spec.Configuration.VirtualMachineInstancesPerNode`. Otherwise, we might get this test trivially passing by not changing any config.